### ISSUE TITLE
Voice: passives and causatives

### ISSUE THREAD

**Issue by LanguageStructure** [UniversalDependencies/docs#998]
Some languages have a passive marker that cooccur with a causative marker: 
Bororo:
_aogwa-do-dy_ 
hide-CAU-PASS
 'caused to hide' (be interrupted).

Since causatives and passives are rarely in complementary distribution, I wonder why they are both voice markers UD. Is it possible to use both without creating a new feature?

**Comment by amir-zeldes** [comment]
This issue may be helpful: https://github.com/unipv-larl/UD4HL/issues/9

**Comment by dan-zeman** [comment]
The de-facto standard solution to this in UD is to define not a new feature, but a language-specific value of the existing feature `Voice=CauPass`; see e.g. [Turkish](https://universaldependencies.org/tr/feat/Voice.html#CauPass).

**Comment by LanguageStructure** [comment]
Thank you! I shall stick to `Voice=CauPass`.

**Comment by Stormur** [comment]
I had to think back to this issue while reading a passage in Dixon's [Basic Linguistic Theory 1](https://academic.oup.com/book/51152) about syntactic terminology (p. 235):

> **Voice** is the traditional term for derivations which reduce valency—passive and now also antipassive. It is preferable to  restrict ‘voice’ to this sense, and not extend it to valency-increasing derivations such as causative and applicative, or to reﬂexive and reciprocal when they change valency. (Some linguists do accord a wider meaning to ‘voice’ and do so in a variety of diﬀerent ways, which may lead to confusion.)

So, this could in fact lead to acknowledge different features for values like `Causative` and `Passive`, based on whether valency is positively or negatively affected, and from the data like for Bororo we do get a confirmation of this (but it is already well known that this happens periphrastically for many languages, e.g. Italian _mi sono fatto vedere_ 'I made see myself', using causative _fare_ 'to make/do' and "reflexive" strategy).

Possible tags could depend on other terms like _(de)transitivizing_ (also used in Dixon's work), maybe also covering possibly morphologically expressed _ambitransitivizing_ strategies (one example, very dear to Dixon, could be Fijian), and others. 

In my opinion this should be seriously considered, and: might a similar feature split be not so drastic to be still part of UD 2?
